hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -497,6 +497,12 @@ int MDMParser::waitFinalResp(_CALLBACKPTR cb /* = NULL*/,
                             if ((r >= 4) && (c != (int)0xFFFFFFFF))  _net.cgi.cell_id  = c;
                             // access technology
                             if (r >= 5) {
+                                if (_dev.dev == DEV_SARA_R410) {
+                                    if (d == static_cast<int>(UbloxSaraCellularAccessTechnology::UBLOX_SARA_RAT_LTE)) {
+                                        d = static_cast<int>(UbloxSaraCellularAccessTechnology::UBLOX_SARA_RAT_LTE_CAT_M1);
```

Comment:
Can we instead move the conversions out of the callbacks and into the actual `getSignalStrength()` where it really matters? We already have the fixup there for G350.